[2018-10-26 18:07:05,831] ERROR in main.get_value Line 67: user_id: 12sd12d1213, response: {"request": {"d1": 0.630868739146, "d2": 2.11659566222, "d3": 1.08099106563, "d4": 2.48976813389, "d5": 2.66392202604, "d6": 2.71439830305, "d7": 1.14912205052, "d8": 2.76907394478, "d9": 5.62600470797, "d10": 5.964746185}, "response": 3.412720422204076}
[2018-10-26 18:07:07,798] ERROR in main.get_value Line 67: user_id: 12sd12d1213, response: {"request": {"d1": 0.630868739146, "d2": 2.11659566222, "d3": 1.08099106563, "d4": 2.48976813389, "d5": 2.66392202604, "d6": 2.71439830305, "d7": 1.14912205052, "d8": 2.76907394478, "d9": 5.62600470797, "d10": 5.964746185}, "response": 3.412720422204076}
[2018-10-26 18:07:15,016] ERROR in main.get_value Line 67: user_id: 12sd12d1213, response: {"request": {"d1": 0.630868739146, "d2": 2.11659566222, "d3": 1.08099106563, "d4": 2.48976813389, "d5": 2.66392202604, "d6": 2.71439830305, "d7": 1.14912205052, "d8": 2.76907394478, "d9": 5.62600470797, "d10": 5.964746185}, "response": 3.412720422204076}
[2018-10-26 18:07:16,205] ERROR in main.get_value Line 67: user_id: 12sd12d1213, response: {"request": {"d1": 0.630868739146, "d2": 2.11659566222, "d3": 1.08099106563, "d4": 2.48976813389, "d5": 2.66392202604, "d6": 2.71439830305, "d7": 1.14912205052, "d8": 2.76907394478, "d9": 5.62600470797, "d10": 5.964746185}, "response": 3.412720422204076}
[2018-10-26 18:07:16,546] ERROR in main.get_value Line 67: user_id: 12sd12d1213, response: {"request": {"d1": 0.630868739146, "d2": 2.11659566222, "d3": 1.08099106563, "d4": 2.48976813389, "d5": 2.66392202604, "d6": 2.71439830305, "d7": 1.14912205052, "d8": 2.76907394478, "d9": 5.62600470797, "d10": 5.964746185}, "response": 3.412720422204076}
[2018-10-26 18:36:33,658] ERROR in main.get_value Line 78: user_id: 5555553, response: {"request": {"d1": 0.630868739146, "d2": 2.11659566222, "d3": 1.08099106563, "d4": 2.48976813389, "d5": 2.66392202604, "d6": 2.71439830305, "d7": 1.14912205052, "d8": 2.76907394478, "d9": 5.62600470797, "d10": 5.964746185}, "response": 3.412720422204076}
[2018-10-26 18:36:42,473] ERROR in main.get_value Line 78: user_id: 5555553, response: {"request": {"d1": 0.0, "d2": 2.11659566222, "d3": 1.08099106563, "d4": 2.48976813389, "d5": 2.66392202604, "d6": 2.71439830305, "d7": 1.14912205052, "d8": 2.76907394478, "d9": 5.62600470797, "d10": 5.964746185}, "response": -1.8974988298370117}
[2018-10-26 18:36:46,355] ERROR in main.get_value Line 78: user_id: 5555553, response: {"request": {"d1": 0.0, "d2": 2, "d3": 2, "d4": 2, "d5": 2, "d6": 2, "d7": 2, "d8": 2, "d9": 2, "d10": 2}, "response": -2.713455809174726}
[2018-10-26 18:36:55,694] ERROR in main.get_value Line 78: user_id: 5555553, response: {"request": {"d1": 2, "d2": 2, "d3": 2, "d4": 2, "d5": 2, "d6": 2, "d7": 2, "d8": 2, "d9": 2, "d10": 2}, "response": 190.1047262588301}
[2018-10-26 18:37:04,231] ERROR in main.get_value Line 78: user_id: 5a5a5a5a5a5a3, response: {"request": {"d1": 2, "d2": 2, "d3": 2, "d4": 2, "d5": 2, "d6": 2, "d7": 2, "d8": 2, "d9": 2, "d10": 2}, "response": 190.1047262588301}
